Cython/Compiler/Dataclass.py

```diff
@@ -81,40 +81,37 @@ def visit_Node(self, node):
         return node
 
 
-class TemplateCode(object):
+class TemplateCode(PyxCodeWriter):
```

Comment:
Hmm. *Is* this a `PyxCodeWriter`, or should it just be *using* one?

Reply:
Somewhere in between maybe?

I refactored it to just use one, which I think works fine